Assets/MRTK/Services/InputSystem/GazeProvider.cs

```diff
@@ -468,56 +480,54 @@ public void OnInputDown(InputEventData eventData)
+
         private IMixedRealityPointer InitializeGazePointer()
         {
-            Profiler.BeginSample("[MRTK] GazeProvider.InitializeGazePointer");
-
-            if (gazeTransform == null)
+            using (InitializeGazePointerPerfMarker.Auto())
             {
-                gazeTransform = CameraCache.Main.transform;
-            }
-
-            Debug.Assert(gazeTransform != null, "No gaze transform to raycast from!");
+                if (gazeTransform == null)
+                {
+                    gazeTransform = CameraCache.Main.transform;
+                }
 
-            gazePointer = new InternalGazePointer(this, "Gaze Pointer", null, raycastLayerMasks, maxGazeCollisionDistance, gazeTransform, stabilizer);
+                Debug.Assert(gazeTransform != null, "No gaze transform to raycast from!");
 
-            if ((GazeCursor == null) &&
-                (GazeCursorPrefab != null))
-            {
-                GameObject cursor = Instantiate(GazeCursorPrefab);
-                MixedRealityPlayspace.AddChild(cursor.transform);
-                SetGazeCursor(cursor);
-            }
+                gazePointer = new InternalGazePointer(this, "Gaze Pointer", null, raycastLayerMasks, maxGazeCollisionDistance, gazeTransform, stabilizer);
 
-            Profiler.EndSample(); // InitializeGazePointer
+                if ((GazeCursor == null) &&
+                    (GazeCursorPrefab != null))
+                {
+                    GameObject cursor = Instantiate(GazeCursorPrefab);
+                    MixedRealityPlayspace.AddChild(cursor.transform);
+                    SetGazeCursor(cursor);
+                }
 
-            return gazePointer;
+                return gazePointer;
+            }
         }
 
+        private static readonly ProfilerMarker RaiseSourceDetectedPerfMarker = new ProfilerMarker("[MRTK] GazeProvider.RaiseSourceDetectec");
+
         private async void RaiseSourceDetected()
         {
-            Profiler.BeginSample("[MRTK] GazeProvider.RaiseSourceDetected");
-
-            await EnsureInputSystemValid();
-
-            if (this == null)
+            using (RaiseSourceDetectedPerfMarker.Auto())
             {
-                // We've been destroyed during the await.
-                Profiler.EndSample(); // RaiseSourceDetected - early exit
-                return;
-            }
+                await EnsureInputSystemValid();
 
-            CoreServices.InputSystem?.RaiseSourceDetected(GazeInputSource);
-            GazePointer.BaseCursor?.SetVisibility(true);
+                if (this == null)
+                {
+                    // We've been destroyed during the await.
+                    return;
+                }
 
-            Profiler.EndSample(); // RaiseSourceDetected
+                CoreServices.InputSystem?.RaiseSourceDetected(GazeInputSource);
+                GazePointer.BaseCursor?.SetVisibility(true);
+            }
         }
 
-        /// <inheritdoc />
```

Comment:
Why's this being removed?

Reply:
accidentally deleted. thanks for the catch!